build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group 'se.fortnox.jbehavesinglescenario'
-version '2.4-SNAPSHOT'
+version '2.4'
```

Comment:
I don't remeber how this was actually deployed to intellij plugin system, but I guess I was running it locally in some way. Perhaps we don't want to have a fixed version in here, but only change it during release?

Reply:
I honestly don't know, I just noticed that in the IntelliJ plugin market it now says "-SNAPSHOT" of the version so I figured this was wrong. Should I change it back?